.prettierrc.js

```diff
@@ -12,6 +12,7 @@ const config = {
     "prettier-plugin-astro",
     "prettier-plugin-organize-imports",
     "prettier-plugin-tailwindcss",
+    "prettier-plugin-svelte",
```

Comment:
```suggestion
    "prettier-plugin-svelte",
    "prettier-plugin-tailwindcss",
```

https://github.com/tailwindlabs/prettier-plugin-tailwindcss?tab=readme-ov-file#compatibility-with-other-prettier-plugins